### DISCUSSION TITLE
Open and close folders in File View

### DISCUSSION THREAD

**Discussion by lolbat** [notaapp/talk#330] [Feature Requests]
I work with a large number of files when I am writing in Nota. Typically they are images that I add to content that I am writing. I might, for a series of articles, have 100+ images that I am working with. 

The Files View show the folder name and then lists any sub folders and images. I can't toggle those folders on/off though so I have many files to scroll through whenever I need to add images to an article. 

This is part of the issue in another discussion entry I added #329

**Comment by astoilkov** [comment]
Hey @lolbat.

Why can't you toggle these folders on/off? Clicking the folder icon should collapse a folder.

Thanks for the feedback.

**Comment by lolbat** [comment]
There is no folder icon. This is in the File View not the Folder View. 

![File View](https://user-images.githubusercontent.com/4564571/198283864-de373f34-ff56-4376-9d73-ed7232524a28.jpeg)

**Reply by astoilkov** [reply]
Sorry. I should have understood that from the first read. Yeah, this makes sense. We will see what we can do.

**Comment by lolbat** [comment]
No worries. Too many similar terms for the sidebars :-)

Even something as simple as double-clicking the name to close it would be great.
